app.rb

```diff
@@ -1,35 +1,64 @@
-require 'net/http'
-require 'uri'
-require 'openssl'
-require 'json'
-require 'redis'
-
-redis = Redis.new(:host => ENV["REDIS_HOST"])
-
-# TODO: Query database for each unique cluster fqdn
-# for each cluster fqdn, append port number, endpoint, and query
-fqdn = ['ganeti-psf.osuosl.bak', 'ganeti-civicrm.osuosl.bak']
-fqdn.each do |name|
+require 'sinatra/base'
+require File.expand_path '../environment.rb', __FILE__
+
+# IAM - a resource usage metric collection and reporting system
+class Iam < Sinatra::Base
+  # require the models, but make sure to do it after the test db is migrated
+  require 'models'
+
+  # load up all the plugins
+  Dir[File.dirname(__FILE__) + '/plugins/*.rb'].each { |file| require file }
```

Comment:
we did make that change, but that wasn't in develop when I branched - I want to keep the scope of this PR pretty narrow, but since this also will relate to how we load plugin rspec, I'll see if I can fix it.